TI-reports/2025/2025-Q3-BEST-WG.md

```diff
@@ -0,0 +1,189 @@
+# 2025 Q3 Best Practices WG
+
+## Overview
+
+### **Sub-groups**
+
+* [Guides](https://github.com/ossf/wg-best-practices-os-developers/tree/main/docs)  
+* [EDU.SIG](https://github.com/ossf/education/)  
+* [Memory Safety SIG](https://github.com/ossf/Memory-Safety)  
+* [OpenSSF Best Practices Badge](https://www.bestpractices.dev/)  
+* [OpenSSF Scorecard](https://github.com/ossf/scorecard)  
+* [Best Practices Badge and Developing Secure Software (LFD121) course](https://github.com/ossf/secure-sw-dev-fundamentals)  
+* [Security Baseline](https://github.com/ossf/security-baseline)  
+* [Web (with SWAG)](https://github.com/w3c-cg/swag/)
+
+## **Activity**
+
+### **Best Practices Badge**
+
+#### **Purpose**
+
+* The Open Source Security Foundation (OpenSSF) Best Practices badge is a way for Free/Libre and Open Source Software (FLOSS) projects to show that they follow best practices. Projects can voluntarily self-certify, at no cost, by using this web application to explain how they follow each best practice.
+
+#### **Status Update**
+
+* Continued maintenance and security updates.   
+* Resolved subtle problem that was preventing ONAP from earning a gold badge.  
+* Extremely high number of requests results in logs growing too large to store. The requests are likely caused by content collection for AI training.
+
+**Up Next**
+
+* Continued maintenance.
+
+### **EDU.SIG**
+
+#### **Purpose**
+
+* Deliver Baseline Secure Software Development Education and Certification to All. Provide access to open and widely available education materials to all learners. Materials will be maximally accessible and easy to consume for all learners.
+
+#### **Status Update**
+
+* LFD121 labs are being translated into Japanese\!  
```

Comment:
This is great! Not directly related to the BEST WG, but I'm curious whether this translation work is being done by a contractor or if this being driven by a community member? The SLSA spec has received a few localization requests (incl. Japanese), and it would be great to continue the trend of having OpenSSF resources be available in multiple languages.